stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 6.2.1
+version: 6.2.2
```

Comment:
Can you update to 6.3.0 since it is a new feature

Reply:
updated